### PR TITLE
AA/join+news

### PR DESCRIPTION
**Pull request by aastha170902**
Changes made to the Resources page and Join us page.

### CODE REVIEW

File: join.html [hunk 1]
```diff
@@ -239,22 +204,20 @@ <h3 class="text-center wdwd">Recruitment for SRM Students</h3>
...
-                non cupidatat skateboard dolor brunch. Food truck quinoa
-                nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua
-                put a bird on it squid single-origin coffee nulla assumenda
-                shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore
-                wes anderson cred nesciunt sapiente ea proident. Ad vegan
-                excepteur butcher vice lomo. Leggings occaecat craft beer
-                farm-to-table, raw denim aesthetic synth nesciunt you probably
-                haven't heard of them accusamus labore sustainable VHS.
+                Being a part of our community is absolutely free of cost. While to acquire an ACM Student Membership a small amount is required to be paid. 
+Some of the perks of being a  member out of many include : <br>
```
Comment (sanskritikhare142) [inline join.html]: Please use prettier to format all the files

File: news.html [hunk 2]
```diff
@@ -52,7 +52,10 @@ <h1>
                 <p>In Computing sharing our progress and resources help our fellow developers in their own endeavours.</p>
             </div>
         </div>
-        <h3 class="wdwd text-center">Our Latest Achievements</h3>
+        <br>
```
Comment (sanskritikhare142) [inline news.html]: `br` is a self-closing tag. Please change the syntax or remove it if it is redundant.

File: join.html [hunk 3]
```diff
@@ -76,17 +76,8 @@ <h1>
             Coming together is a beginning; keeping together is progress;
             working together is success.
           </p>
-          <h3 class="gradient">
-            Recruiting Now! Register Yourself<span
-              ><a href="https://cutt.ly/acm-w-srmist" target="_blank">
-                <svg
```
Comment (sanskritikhare142) [inline join.html:82]: I think removing this `svg` tag would be throwing some errors. The closing tag is still present and the image too. So the opening tag should also be present. Did you check the console for error message?

File: join.html [hunk 4]
```diff
@@ -239,22 +204,20 @@ <h3 class="text-center wdwd">Recruitment for SRM Students</h3>
...
-                nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua
-                put a bird on it squid single-origin coffee nulla assumenda
-                shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore
-                wes anderson cred nesciunt sapiente ea proident. Ad vegan
-                excepteur butcher vice lomo. Leggings occaecat craft beer
-                farm-to-table, raw denim aesthetic synth nesciunt you probably
-                haven't heard of them accusamus labore sustainable VHS.
+                Being a part of our community is absolutely free of cost. While to acquire an ACM Student Membership a small amount is required to be paid. 
+Some of the perks of being a  member out of many include : <br>
+1. Access to the full ACM Digital Library. <br>
```
Comment (sanskritikhare142) [inline join.html]: Avoid using `<br/>` tag for changing lines. Rather wrap it in a fluid container. Also check if the text is responsive.

**Review by sanskritikhare142 — CHANGES_REQUESTED** [review]

File: join.html (hunk 1 above)
Comment (aastha170902) [reply join.html]: okay

File: news.html (hunk 2 above)
Comment (aastha170902) [reply news.html]: I'll look into it.

File: join.html (hunk 3 above)
Comment (aastha170902) [reply join.html:82]: No I didn't, I'll do that

File: join.html (hunk 4 above)
Comment (aastha170902) [reply join.html]: okay okay

File: join.html [hunk 5]
```diff
@@ -1,317 +1,192 @@
...
+            <h2 class="mb-0 collapsed" data-toggle="collapse" data-target="#collapseFour" aria-expanded="false"
+              aria-controls="collapseThree">
+              Do we have to pay for ACM Membership to join the community?
+            </h2>
+          </div>
+          <div id="collapseFour" class="collapse" aria-labelledby="headingThree" data-parent="#accordionExample">
+            <div class="card-body">
+              Being a part of our community is absolutely free of cost. While to acquire an ACM Student Membership a
+              small amount is required to be paid.
+              Some of the perks of being a member out of many include : <br>
```
Comment (sanskritikhare142) [inline join.html]: Please check the syntax of `br` tag.

**Review by sanskritikhare142 — CHANGES_REQUESTED** [review]

File: join.html (hunk 5 above)
Comment (aastha170902) [reply join.html]: done

**Pull request merged**